vita3k/gui/src/game_context_menu.cpp

```diff
@@ -188,163 +180,147 @@ void game_context_menu(GuiState &gui, HostState &host) {
+
+    // Context Dialog
+    if (!context_dialog.empty()) {
+        ImGui::SetNextWindowPos(ImVec2(0, 0), ImGuiCond_Always);
+        ImGui::SetNextWindowSize(display_size, ImGuiCond_Always);
+        ImGui::Begin("##context_dialog", nullptr, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoBackground | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoSavedSettings);
+        ImGui::SetNextWindowPosCenter();
+        ImGui::BeginChild("##context_dialog_child", WINDOW_SIZE, true, ImGuiWindowFlags_NoMove | ImGuiWindowFlags_NoDecoration | ImGuiWindowFlags_NoSavedSettings);
+        // Update History
+        if (context_dialog == "history") {
+            ImGui::SetCursorPos(ImVec2(20.f * scal.x, BUTTON_SIZE.y));
+            if (ImGui::ListBoxHeader("##update_history_list", ImVec2(WINDOW_SIZE.x - (40.f * scal.x), WINDOW_SIZE.y - (BUTTON_SIZE.y * 2.f) - (25.f * scal.y)))) {
+                for (auto u = 0; u < update_history_infos[host.io.title_id].first.size(); u++) {
+                    ImGui::SetWindowFontScale(1.8f);
+                    ImGui::TextColored(GUI_COLOR_TEXT, "Version %.2f", update_history_infos[host.io.title_id].first[u]);
+                    ImGui::SetWindowFontScale(1.4f);
+                    ImGui::PushTextWrapPos(WINDOW_SIZE.x - (80.f * scal.x));
+                    ImGui::TextColored(GUI_COLOR_TEXT, "%s\n", update_history_infos[host.io.title_id].second[u].c_str());
+                    ImGui::PopTextWrapPos();
+                    ImGui::TextColored(GUI_COLOR_TEXT, "\n");
+                }
+                ImGui::ListBoxFooter();
+            }
+            ImGui::SetWindowFontScale(1.4f * scal.x);
+            ImGui::SetCursorPos(ImVec2((WINDOW_SIZE.x / 2.f) - (BUTTON_SIZE.x / 2.f), WINDOW_SIZE.y - BUTTON_SIZE.y - (22.f * scal.y)));
+        } else {
+            // Delete Data
+            if (gui.game_selector.icons.find(host.io.title_id) != gui.game_selector.icons.end()) {
+                ImGui::SetCursorPosX((WINDOW_SIZE.x / 2.f) - (ICON_SIZE.x / 2.f));
+                ImGui::Image(gui.game_selector.icons[host.io.title_id], ICON_SIZE);
+            }
+            ImGui::SetWindowFontScale(1.4f * scal.x);
+            ImGui::SetCursorPosX((WINDOW_SIZE.x / 2.f) - (ImGui::CalcTextSize(host.game_short_title.c_str()).x / 2.f));
+            ImGui::TextColored(GUI_COLOR_TEXT, host.game_short_title.c_str());
+            std::string ask_delete = context_dialog == "save" ? "Do you really want to delete this save data for this application?" : "Do you really want to delete this application?";
+            ImGui::SetCursorPos(ImVec2(WINDOW_SIZE.x / 2 - ImGui::CalcTextSize(ask_delete.c_str()).x / 2, (WINDOW_SIZE.y / 2) + 10));
+            ImGui::TextColored(GUI_COLOR_TEXT, ask_delete.c_str());
+            if (context_dialog == "game") {
+                if (ImGui::IsItemHovered())
+                    ImGui::SetTooltip("Deleting a game may take a while!\nDepending on the size of the games and your hardware.");
```

Comment:
```suggestion
                    ImGui::SetTooltip("Deleting a game may take a while\ndepending on its size and your hardware.");
```

Reply:
Oh for this is other dev have write it first time i untouched here

Reply:
Ok, change it anyway because it's incorrect English.

Reply:
oki is done